typescript/sdk/src/providers/MultiProtocolProvider.ts

```diff
@@ -39,10 +40,12 @@ export interface MultiProtocolProviderOptions {
 export class MultiProtocolProvider<
   MetaExt = {},
 > extends ChainMetadataManager<MetaExt> {
-  protected readonly providers: ChainMap<ProviderMap<TypedProvider>> = {};
+  // Chain name -> provider type -> provider
+  protected readonly providers: ChainMap<ProviderMap<TypedProvider>>;
```

Comment:
Do we expect to have multiple different types of providers for a single chain? e.g. viem and ethers

Reply:
I don't necessarily expect it but it is supported. 
One scenario where we'd want that is in a UI that uses both viem and our SDK (b.c. our SDK uses ethers v5)